ArmPkg/ArmPkg.dsc

```diff
@@ -119,7 +119,7 @@
   ArmPkg/Library/SemihostLib/SemihostLib.inf
   ArmPkg/Library/ArmExceptionLib/ArmExceptionLib.inf
 
-  ArmPkg/Drivers/CpuDxe/CpuDxe.inf
+  UefiCpuPkg/CpuDxeArm/CpuDxe.inf

```

Comment:
This is odd to have ArmPkg build a component that lives in UefiCpuPkg. I would expect UefiCpuPkg.dsc to build all the components that live within its package. I understand that the library dependencies are still within ArmPkg, but it doesn't seem to me that there is much value in moving the driver if the libraries aren't correctly dealt with. Untangling the ArmPkg libraries is a monumental task, but it is also the right direction to go

Reply:
Decoupled UefiCpuPkg/CpuDxeArm/CpuDxe.inf with ArmPkg/ArmPkg.dsc in latest commit.